src/components/Link.js

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import { Link as GatsbyLink } from "gatsby"
+import PropTypes from "prop-types"
+
+const Link = ({ isExternal = false, url, children, ...props }) => {
+  if (isExternal || /^http/.test(url)) {
```

Comment:
This is interesting---any reason why we require `isExternal` to be passed vs just always inferring it from the URL?

Reply:
No reason...I wrote that prop before I decided we can just infer it and just didn't take it out after the fact. 😬 Will clean it up!!